common/types.go

```diff
@@ -0,0 +1,25 @@
+package common
+
+import (
+	"net/url"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/livepeer/go-livepeer/net"
+)
+
+type Broadcaster interface {
+	Address() ethcommon.Address
+	Sign([]byte) ([]byte, error)
+}
+
+type OrchestratorPool interface {
```

Comment:
It *feels* like we had earlier problems with the OrchestratorPool in `common`  - hence why it was in `net` - but if it's not leading to any issues right now, then this is probably a better place for it.